tests/test_moe/test_moe_ep_tp.py

```diff
@@ -21,7 +132,12 @@ def run_test(rank: int, world_size: int, port: int, num_experts: int, batch_size
     local_model = SparseMLP(num_experts=num_experts, hidden_size=dim, intermediate_size=dim * 2)
     MOE_MANAGER.__init__()
     MOE_MANAGER.setup(seed, parallel="EP")
-    ep_model = SparseMLP(num_experts=num_experts, hidden_size=dim, intermediate_size=dim * 2)
+    ep_model = SparseMLP(
+        num_experts=num_experts,
+        hidden_size=dim,
+        intermediate_size=dim * 2,
+        create_hierarchical_group=lambda group: create_ep_hierarchical_group(group, num_node=1, nproc_per_node=2)
```

Comment:
Can we auto set this?

Reply:
This can be auto-set if `torchrun` is used.

> `LOCAL_WORLD_SIZE` - The local world size (e.g. number of workers running locally); equals to `--nproc-per-node` specified on `torchrun`. 

9795011 resolves this.